rust-runtime/aws-smithy-types/src/byte_stream/bytestream_util.rs

```diff
@@ -238,407 +244,3 @@ enum State {
         bytes_left: u64,
     },
 }
-
-impl http_body_0_4::Body for PathBody {
```

Comment:
Moving this isn't breaking b/c it's only an impl and it already required the `http-body-0-4-x` feature.